ui/app/ducks/swaps/swaps.js

```diff
@@ -88,6 +99,32 @@ const slice = createSlice({
     setToToken: (state, action) => {
       state.toToken = action.payload
     },
+    swapCustomGasModalPriceEdited: (state, action) => {
+      state.customGas = {
+        ...state.customGas,
+        price: action.payload,
+      }
```

Comment:
We're using `immer`, so we can mutate the state directly and trust it to handle updating it immutably:

e.g.:
```suggestion
      state.customGas.price = action.payload
```

This applies to the others as well

Reply:
9b22e67f1